board/piksiv3/rootfs-overlay/etc/init.d/S86skylark0_setup

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+name="skylark_setup"
+
+start() {
+  # Setup DNS
+  # TODO (mookerji): Move to a template command?
+  rm -f /etc/resolv.conf
+  echo "nameserver 8.8.8.8" > /etc/resolv.conf
+  date -s '2017-03-09 16:14:00'
+
+  # Roll your own formatted UUID for Skylark client
```

Comment:
Can we not have the init put that into the config file system for us? as `/cfg/device_uid` or whatever - this is the device uid not particularly a skylark thing. I think that would be great to get in there - just don't think we should be doing it in our application.